spec/consensus/proposer-based-timestamp/README.md

```diff
@@ -67,22 +68,17 @@ is bounded by `MSGDELAY`.
 This assumption is restricted to `Proposal` messages, broadcast by proposers.
 
 `PRECISION` and `MSGDELAY` are consensus parameters, shared by all validators,
-that define whether the timestamp of a block is acceptable.
-Let `t` be the time, read from its local clock, at which a validator
-receives, for the first time, a proposal with timestamp `ts`:
+that define whether the timestamp of a block is acceptable,
+according with the introduced `timely` predicate.
 
-- **[Time-Validity]** The proposed timestamp `ts` received at local time `t`
-is accepted if it satisfies the **timely** predicate:
-	> `ts - PRECISION <= t <= ts + MSGDELAY + PRECISION`
+In short, let `proposalReceiveTime` be the time, read from its local clock, at
+which a validator receives a proposal for a `block` with timestamp `block.time`.
+The proposed timestamp `block.time` can be accepted if
 
-The left inequality of the *timely* predicate establishes that proposed timestamps
-should be in the past, when adjusted by the clocks `PRECISION`.
-The right inequality of the *timely* predicate establishes that proposed timestamps
-should not be too much in the past, more precisely, not more than `MSGDELAY` in the past,
-when adjusted by the clocks `PRECISION`.
```

Comment:
This paragraph was removed since the same reasoning is presented in the system model (and it belongs there).